src/python/pants/backend/python/typecheck/mypy/rules_integration_test.py

```diff
@@ -425,6 +430,71 @@ def test_works_with_python38(rule_runner: RuleRunner) -> None:
     assert "Success: no issues found" in result[0].stdout.strip()
 
 
+@skip_unless_python27_and_python3_present
```

Comment:
Do we not support any python 3 under 3.6? Just because we use `CPython>=3.6` below.

Reply:
Ah, it looks like 3.5 is EOL. That's fine, then.